docs/source/create/entities/create-fountain.rst

```diff
@@ -0,0 +1,93 @@
+#####################################
+Tutorial: Create a Smoke Fountain
+#####################################
+
+In this tutorial, you will learn how particle entities work by creating a smoke fountain that emits multiple colors. Particle entities are used to create effects made up of many small particles, such as fire, smoke, or falling leaves.
```

Comment:
Fire and smoke are similar concepts. So let's replace with something different :)

```suggestion
In this tutorial, you will learn how particle entities work by creating a smoke fountain that emits multiple colors. Particle entities are used to create effects made up of many small particles, such as smoke, confetti, or falling leaves.
```